static/src/javascripts/bootstraps/commercial.js

```diff
@@ -20,150 +20,153 @@ import { init as initThirdPartyTags } from 'commercial/modules/third-party-tags'
 
-        catchErrorsWithContext(
-            [
-                [
-                    moduleName,
-                    function pushAfterComplete() {
-                        const result = moduleInit();
-                        modulePromises.push(result);
-                    },
-                ],
-            ],
-            {
-                feature: 'commercial',
-            }
-        );
-    });
+		catchErrorsWithContext(
+			[
+				[
+					moduleName,
+					function pushAfterComplete() {
+						const result = moduleInit();
+						modulePromises.push(result);
+					},
+				],
+			],
+			{
+				feature: 'commercial',
+			},
+		);
+	});
 
-    return Promise.all(modulePromises);
+	return Promise.all(modulePromises);
 };
 
 export const bootCommercial = () => {
-    // Init Commercial event timers
-    EventTimer.init();
+	// Init Commercial event timers
+	EventTimer.init();
 
-    catchErrorsWithContext(
-        [
-            [
-                'ga-user-timing-commercial-start',
-                function runTrackPerformance() {
-                    EventTimer.get().trigger('commercialStart');
-                },
-            ],
-        ],
-        {
-            feature: 'commercial',
-        }
-    );
+	catchErrorsWithContext(
+		[
+			[
+				'ga-user-timing-commercial-start',
+				function runTrackPerformance() {
+					EventTimer.get().trigger('commercialStart');
+				},
+			],
+		],
+		{
+			feature: 'commercial',
+		},
+	);
 
-    // Stub the command queue
-    window.googletag = {
-        cmd: [],
-    };
+	// Stub the command queue
+	window.googletag = {
+		cmd: [],
+	};
 
-    return loadHostedBundle()
-        .then(loadModules)
-        .then(() => {
-            catchErrorsWithContext(
-                [
-                    [
-                        'ga-user-timing-commercial-end',
-                        function runTrackPerformance() {
-                            EventTimer.get().trigger('commercialEnd');
-                        },
-                    ],
-                ],
-                {
-                    feature: 'commercial',
-                }
-            );
-        })
-        .catch(err => {
-            // report async errors in bootCommercial to Sentry with the commercial feature tag
-            reportError(
-                err,
-                {
-                    feature: 'commercial',
-                },
-                false
-            );
-        });
+	return loadHostedBundle()
+		.then(loadModules)
+		.then(() => {
+			catchErrorsWithContext(
+				[
+					[
+						'ga-user-timing-commercial-end',
+						function runTrackPerformance() {
+							EventTimer.get().trigger('commercialEnd');
+						},
+					],
+				],
+				{
+					feature: 'commercial',
+				},
+			);
+		})
+		.catch((err) => {
+			// report async errors in bootCommercial to Sentry with the commercial feature tag
+			reportError(
+				err,
+				{
+					feature: 'commercial',
+				},
+				false,
+			);
+		});
```

Comment:
More of an observation - are these formatting rules coming from the project settings (my VS Code Prettier doesn't enforce these rules for some reason) or an individual developer's prettier setup?

Either way we should be consistent with formatting across the team

Reply:
It should be a mix of `.editorConfig` and `eslint`. This project has not had as much love as others, and maybe DevX could run a formatting/linting on all JS files?